security/docker/Dockerfile.node-agent-k8s

```diff
@@ -1,4 +1,5 @@
-FROM istionightly/base_debug
+FROM ubuntu:16.04
```

Comment:
I know this is WIP, but just a note, I think we switched this to the debug image intentionally because we had issues with not having debug tools on the container

Reply:
The user guide https://istio.io/docs/tasks/security/vault-ca/ succeeds on istio-release-1.2-20190529-09-16. This PR is closed for now. Will reopen if Node Agent crashes again.